### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by zmwangx**
Four unrelated commits.

Most notable is the second one: "Strip and collapse consecutive whitespace in user commands".

This branch is based on #90. Need to merge that first.

### CODE REVIEW

**Comment by zmwangx** [comment]
Rebased.

File: googler [hunk 1]
```diff
@@ -917,8 +918,8 @@ def fetch_results():
     return results, urlindex
 
 
-def show_omniprompt():
-    """Show the search or navigation omniprompt."""
+def read_command():
```
Comment (jarun) [inline googler]: Please change the name to `read_next_command`.

Comment (jarun) [inline googler]: Or just leave it, I'll do.

**Comment by jarun** [comment]
Thank you!

**Pull request merged**